client/src/+assets/user-publish-steps/basic-info.tsx

```diff
@@ -30,21 +33,26 @@ export const BasicInfoStep = (props: BasicInfoProps) => {
   const [descriptionInputError, setDescriptionInputError] = useState('')
   const [subscriptionInputError, setSubscriptionInputError] = useState('')
 
-  const { userSubscriptions, getCurrentUserSubscription} = useContext(User)
+  const { userSubscriptionsStatus, userSubscriptions, getCurrentUserSubscription } =
+    useContext(User)
   const [tiers, setTiers] = useState<string[]>([])
-  const subscriptionErrorText = "You don't have any current subscription. Only users with a subscription are allowed to publish"
+  const subscriptionErrorText =
+    "You don't have any current subscription. Only users with a subscription are allowed to publish"
+  const { isLoadingSDK } = Catalog.useNevermined()
 
   useEffect(() => {
-
-    if (!getCurrentUserSubscription()) {
+    if (
+      !userSubscriptionsStatus.isLoading &&
+      userSubscriptionsStatus.hasLoaded &&
```

Comment:
Fix for the async loading of the subscription list
<img width="901" alt="Screenshot 2022-09-26 at 16 29 08" src="https://user-images.githubusercontent.com/1643399/192318170-c3a678fd-f2ca-4ec8-89ef-a2496a12fda5.png">